CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Here is a template for new release sections
 - Verification of the SoC values after simulation for any physically infeasible values with `E4.verify_state_of_charge` and added tests for this function (#739)
 - Explanation in MVS_parameters_list.csv on how to deactivate the RES constraint. (#752)
 - Add the new parameter `scenario_description` to input files and docs and a section in autoreport describing the scenario being simulated (#722)
+- The parameters`fixed_losses_relative` and `fixed_losses_absolute` were added. It is now possible to model a stratified thermal energy storage. The usage of this new component has been tested and documented (#718)
```

Comment:
Can you descripe if the parameters are optional/required? Did you add pytests and benchmark tests for the new feature?

Also: Did you change any functions for this (yes, you did, A1, C0 and D1, state explicitly the functions you changed below the "changed" section).

```suggestion
- The parameters `fixed_losses_relative` and `fixed_losses_absolute` were added. It is now possible to model a stratified thermal energy storage. The usage of this new component has been tested and documented (#718)
```

Reply:
The rest of your suggestions I've implemented with commit https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/2090e8d17a1cd526d31e9f2fd545799904154f22